src/commcare_cloud/commands/ansible/run_module.py

```diff
@@ -141,7 +137,7 @@ def run_ansible_module(environment, ansible_context, inventory_group, module, mo
     ask_vault_pass = include_vars and public_vars.get('commcare_cloud_use_vault', True)
     if ask_vault_pass:
         cmd_parts += ('--vault-password-file=/bin/cat',)
-    cmd_parts += get_common_ssh_args(environment, use_pem)
+    cmd_parts = get_common_ssh_args(cmd_parts, environment, use_factory_auth=factory_auth)
```

Comment:
I'm not sure I understand why `cmd_parts` needs to get passed in; it looks to me that this change just pushes the `+=`ing to inside that function, which is (for me) harder to reason about. In addition, I _think_ I'm seeing a mistake there that makes `cmd_parts` get totally overridden when you're using factory auth (?)

Reply:
Fixed it based on our convo here:
https://github.com/dimagi/commcare-cloud/pull/1702/commits/f70a5f2d756f90ed5d88422d9e4af563fdb2fe46